### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by paillave** [dotnet/roslyn#48322] [General] [answered]
Just to mention that the wiki disappeared from github. The URL was this one: https://github.com/dotnet/roslyn/wiki

It was very interesting as there was all the documentation about how to handle everything to execute code and scripts (it was at this url: https://github.com/dotnet/roslyn/wiki/Scripting-API-Samples)

There is some documentation about the SDK for code analysis in here, https://docs.microsoft.com/en-us/dotnet/csharp/roslyn-sdk/
but was I need is the documentation that was previously here about how to run scripts.

**Comment by PathogenDavid** [comment, marked as answer]
It's in the repo now, which allows people to submit pull requests to update it. https://github.com/dotnet/roslyn/tree/master/docs/wiki ([Permalink](https://github.com/dotnet/roslyn/tree/374ee3184a64ca07867320668ec4b1162daa2aa4/docs/wiki))

The page you're referring to can be found here: https://github.com/dotnet/roslyn/blob/master/docs/wiki/Scripting-API-Samples.md
